lib/pry/pry_class.rb

```diff
@@ -161,6 +161,7 @@ def self.final_session_setup
   #   Pry.start(Object.new, :input => MyInput.new)
   def self.start(target=nil, options={})
     return if ENV['DISABLE_PRY']
+    raise 'Your configuration has pry set to fail when called, you can change this by unsetting FAILY_PRY' if ENV['FAIL_PRY']
```

Comment:
1. s/pry/Pry/
2. Could you please try to wrap this line so it doesn't look that long (preferably up to 80 characters). It's important to see the condition you're checking and at the moment it's hidden.

Otherwise, looks good :+1: